addons/knowledge/models/knowledge_article.py

```diff
@@ -859,6 +859,8 @@ def article_create(self, title=False, parent_id=False, is_private=False):
                 'body': Markup('<h1>%s</h1>') % title,
```

Comment:
Looks like this one does not have the same formatting (notably the "oe-hint" class).
Could we make it consistent or is that intended?